plugin/src/main/kotlin/org/gradle/github/dependency/extractor/internal/GitHubRepositorySnapshotBuilder.kt

```diff
@@ -74,6 +76,13 @@ class GitHubRepositorySnapshotBuilder(
         )
     }
 
+    private fun manifestName(config: ResolvedConfiguration): String {
+        if (config.identityPath != null) {
+            return "project ${config.identityPath}"
+        }
+        return "build ${config.buildPath}"
+    }
```

Comment:
This is an interesting way of differentiating names. Given these names may be rendered in the GitHub UI, might it be good to capitalize the first letters of `Project` and `Build`?